src/averageAges.js

```diff
@@ -14,12 +14,32 @@
  *
  * @return {number}
  */
+
+function calculateAveragePeopleAge(people) {
+  const ages = people.map(person => person.died - person.born);
+
+  const averageAge = ages.reduce((x, y) => x + y, 0) / ages.length;
+
+  return averageAge;
```

Comment:
There is no need to create a variable to return it on the following line. You can create a variable for sum, for example, to improve readability and return this sum divided by the length